tests/test_multiple_inheritance.py

```diff
@@ -53,15 +53,174 @@ def __init__(self, i, j):
     assert mt.bar() == 4
 
 
-def test_multiple_inheritance_error():
-    """Inheriting from multiple C++ bases in Python is not supported"""
+def test_multiple_inheritance_python():
     from pybind11_tests import Base1, Base2
 
-    with pytest.raises(TypeError) as excinfo:
-        # noinspection PyUnusedLocal
-        class MI(Base1, Base2):
-            pass
-    assert "Can't inherit from multiple C++ classes in Python" in str(excinfo.value)
+    class MI1(Base1, Base2):
+        def __init__(self, i, j):
+            Base1.__init__(self, i)
+            Base2.__init__(self, j)
```

Comment:
What kind of error is raised if someone tries to use a C++ base method but accidentally forgot to call `__init__` for that C++ base? Would be nice to cover it with a test.